libs/edssser/edssser.go

```diff
@@ -152,7 +140,7 @@ AvgTime %s
 	)
 }
 
-func (ss *EDSsser) put(ctx context.Context, t *testing.T) (time.Duration, error) {
+func (ss *EDSsser) put(ctx context.Context, t *testing.T, height int) (time.Duration, error) {
```

Comment:
Why not `uint64` ? I know it's coming from `ss.config.WriteFrom` buuuut this sounds buggy due to cast to a smaller integer (in theory 😉 )

Reply:
So bette to change `ss.config.WriteFrom` and `ss.config.EDSWrites` to `uint64`

Reply:
The Edssser is minor side tool, that was written in low-effort and not worth a refactoring. I would rather avoid adding more changes to this already large PR

Reply:
IMO on this scale changing `int` -> `uint64` is a small addition, `assert` -> `require` is already here.